test/jest/victory-scatter/helper-methods.test.js

```diff
@@ -0,0 +1,78 @@
+/* eslint no-unused-expressions: 0 */
```

Comment:
This could probably be removed after implementing the `notCalled` / `toHaveBeenCalled()` change recommendation by @scottrippey.